src/services/word.service.ts

```diff
@@ -31,17 +36,27 @@ export class WordService {
     )
   }
 
-  async get(): Promise<Partial<IWord>[]> {
+  /** Get words by given query */
+  async get(query: GetWordQueryDTO): Promise<Partial<IWord>[]> {
     return (
       await this.deprecatedWordModel
-        .find()
-        .sort({
-          createdAt: -1,
-        })
+        .find(this.getWordQueryFactory.toFind(query))
+        .sort(this.getWordQueryFactory.toSort())
+        .limit(query.limit)
```

Comment:
should have a maximum limit that I return for the API not handling too much data.

Reply:
handled with comment